website/content/labs/lab9.md

```diff
@@ -160,15 +160,15 @@ Now to generate the documentation, run:
 
 And it will generate the documentation in html! To see it, go to the html directory it generated and click on index.html. The Classes tab should now include your segment and circle types. Make sure that all your functions appear there!
 
-Before continuing, make sure to add and commit the additional segment/circle files you added.
+Before continuing, make sure to add and commit the additional segment/circle files you added. Do not add or commit the automatically-generated `docs/html/` and `docs/latex/` directories!
```

Comment:
Should we tell them to create a `.gitignore` file for this?

Reply:
There is actually already a .gitignore file in the files they will pull from upstream, but I've clarified this in the lab.